pkg/registry/registry_to_model.go

```diff
@@ -0,0 +1,232 @@
+package registry
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/operator-framework/operator-registry/internal/model"
+	"github.com/operator-framework/operator-registry/internal/property"
+)
+
+func ConvertRegistryBundleToModelBundles(b *Bundle) ([]model.Bundle, error) {
+	var bundles []model.Bundle
+	desc, err := b.csv.GetDescription()
+	if err != nil {
+		return nil, fmt.Errorf("Could not get description from bundle CSV:%s", err)
+	}
+	i, err := b.csv.GetIcons()
+	if err != nil {
+		return nil, fmt.Errorf("Could not get icon from bundle CSV:%s", err)
+	}
+	mIcon := &model.Icon{
+		MediaType: "",
+		Data:      []byte{},
+	}
+	if len(i) > 0 {
+		mIcon.MediaType = i[0].MediaType
+		mIcon.Data = []byte(i[0].Base64data)
+	}
+
+	pkg := &model.Package{
+		Name:        b.Annotations.PackageName,
+		Description: desc,
+		Icon:        mIcon,
+		Channels:    make(map[string]*model.Channel),
+	}
+	mb, err := registryBundleToModelBundle(b)
+	mb.Package = pkg
+	if err != nil {
+		return nil, err
+	}
+	for _, ch := range extractChannels(b.Annotations.Channels) {
```

Comment:
One thing we need to discuss here is that even though default channel is not required in the annotations. However, users do use it to update the default channel if they include it. Do we still want to reserve that behavior or we are saying we will ignore the default channel info in annotations from now on and if they want to change default channel, they will need to edit the JSON files manually?

Reply:
I think the decision was to go with the following: 
1) When the first bundle is added to the package, if default channel isn't specified, pick one and assign it as default (similar to what happens in the db now) 
2) If you want to update the default channel, the workflow of expressing that intent with specifying the new default channel in a bundle you're adding is outdated, and we'll ignore all further metadata about default channels starting from the second bundle in the package. If you want to edit the default channel, you must manually edit it in the configs.

Reply:
1. The current behavior is we pick the first channel in lexicographically order. So do we want to reserve that behavior?
2. Sure. That's fine to me but then that's a change from existing `index add` command. So make sure we document it somewhere.

Reply:
1. Yes we do. 
2. Sure thing.